FL/x.H

```diff
@@ -82,7 +82,7 @@ typedef ulong Fl_Offscreen;
   fl_push_no_clip()
 #    define fl_end_offscreen() \
   fl_pop_clip(); fl_window = _sw; _ss->set_current(); \
-  if (!_sgc) XFreeGC(fl_display, fl_gc); fl_gc = _sgc
+  if (!_sgc) { XFreeGC(fl_display, fl_gc); } fl_gc = _sgc
```

Comment:
I would prefer to spread the macro over more lines to make the intention clear.

Reply:
I thought macros were passé with C++.  I tried to go with the spirit of whoever created that thing.  I would spread it out, a statement per line, or replace it with a non-macro implementation.

Reply:
I know and agree. This is old code that abuses macros by hiding the creation of _sw and _ss in fl_begin_offscreen(). To stay backwards compatible, we need to keep the macro, but a better way would be some abstract type that is returned by a better `fl_begin_offscreen()`.